spec/inputs/kinesis_spec.rb

```diff
@@ -77,6 +121,19 @@
     expect(kinesis_with_latest.kcl_config.get_kinesis_credentials_provider.getClass.to_s).to eq("com.amazonaws.auth.DefaultAWSCredentialsProviderChain")
   end
 
+  subject!(:kinesis_with_valid_additional_kcl_options) { LogStash::Inputs::Kinesis.new(config_with_valid_additional_kcl_options) }
+
+  it "configures the KCL" do
+    kinesis_with_latest.register
+    expect(kinesis_with_latest.kcl_config.applicationName).to eq("my-processor")
+    expect(kinesis_with_latest.kcl_config.streamName).to eq("run-specs")
+    expect(kinesis_with_latest.kcl_config.regionName).to eq("ap-southeast-1")
+    expect(kinesis_with_latest.kcl_config.initialLeaseTableReadCapacity).to eq(25)
+    expect(kinesis_with_latest.kcl_config.initialLeaseTableWriteCapacity).to eq(100)
+    expect(kinesis_with_latest.kcl_config.kinesisEndpoint).to eq("http://localhost")
```

Comment:
This expectation does not match the value in the config defined earlier

Reply:
s/www//g